examples/swap.yml

```diff
@@ -40,3 +40,4 @@ services:
 trust:
   org:
     - linuxkit
+    - library
```

Comment:
I think we generally leave line feeds at the end of files. (I think its only your editor that doesn't, so everyone else will keep reverting this...)